doc/reference/peripherals/entropy.rst

```diff
@@ -6,6 +6,10 @@ Entropy
 Overview
 ********
 
+The entropy API provides functions to retrieve entropy values from
+entropy hardware present on the platform. The entropy APIs are provided
+for use by the random subsystem and cryptographic services. They
+are not intended to replace random number generation functions.
```

Comment:
I would suggest: "They are not suitable to be used as random number generation functions."